content/docs/instrumenting/exporters.md

```diff
@@ -118,6 +118,7 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
 
 ### Other monitoring systems
    * [Akamai Cloudmonitor exporter](https://github.com/ExpressenAB/cloudmonitor_exporter)
+   * [Alibaba Cloud Exporter](https://github.com/aylei/aliyun-exporter)
```

Comment:
This looks to be a Cloudmonitor exporter, rather than Alibaba generally

Reply:
How about "Alibaba Cloudmonitor exporter"?

Reply:
Sure